astrobee/config/communications/comms_bridge.config

```diff
@@ -83,7 +88,7 @@ links = {
     relay_backward = {
     },
     relay_both = {
-      {in_topic = "gnc/ekf"},
+      {in_topic = "gnc/ekf", rate=1},
```

Comment:
When we need remote /gnc/ekf we usually want the full rate. I'll likely remove this default rate from the configuration since Katie's on leave.

Reply:
I can do that! Any topic that you usually want to define the rate from the default config?

Reply:
Thank you! Not right now. The rate will likely be useful for custom user topics. But that can be adjusted as needed.

Reply:
ok done